### PR TITLE
866 MenuBar Re-design Changes

### LINKED ISSUE #866: [ReBrand] Menu Bar Design
OLD: https://www.figma.com/file/fgBV6T1rkYBvzQYzwwUnFt/Bioscience-hub-3?type=design&node-id=25307-288582&mode=design&t=KzyfQ8LCJIb2yePc-0

NEW: https://www.figma.com/file/fgBV6T1rkYBvzQYzwwUnFt/Bioscience-hub-3?type=design&node-id=26066-38862&mode=design&t=SyyhjHjJoaDgaPn5-0

### PR DESCRIPTION
**Pull request by amlan18041996**
Fix #866 

Test URLs:
- Before: https://main--danaher-ls-aem--hlxsites.hlx.page/
- After: https://866-rebrand-menu-bar-design--danaher-ls-aem--hlxsites.hlx.page/

### CODE REVIEW

**Comment by rgravitvl** [comment]
HI @amlan18041996 ,

Could you please check below issue?. It loads incorrect menu on initial click.
Steps to reproduce: 
1. Open home page from branch
2. Scroll down for hamburger icon in menu
3. Click on hamburger icon
4. You will below screen shot

image

**Review by rgravitvl — CHANGES_REQUESTED** [review]
Already commented

**Review by davenichols-DHLS — APPROVED** [review]
I like it - going to review with Anthony and then will push.

**Comment by davenichols-DHLS** [comment]
> * https://866-rebrand-menu-bar-design--danaher-ls-aem--hlxsites.hlx.page/

@rgravitvl  and @amlan18041996  - this looks like it is fixed now?  @rgravitvl  - are you happy to approve?

File: blocks/header/header.js [hunk 1]
```diff
@@ -443,7 +458,7 @@ function buildSearchBlock(headerBlock) {
     'aria-controls': 'mega-menu-icons',
     'data-collapse-toggle': 'mega-menu-icons',
   });
-  hamburgerIcon.innerHTML = '<svg xmlns=\'http://www.w3.org/2000/svg\' aria-hidden=\'true\' viewBox=\'0 0 24 24\' fill=\'currentColor\' class=\'w-8 h-8\'><path fill-rule="evenodd" d="M3 6.75A.75.75 0 0 1 3.75 6h16.5a.75.75 0 0 1 0 1.5H3.75A.75.75 0 0 1 3 6.75zM3 12a.75.75 0 0 1 .75-.75h16.5a.75.75 0 0 1 0 1.5H3.75A.75.75 0 0 1 3 12zm0 5.25a.75.75 0 0 1 .75-.75h16.5a.75.75 0 0 1 0 1.5H3.75a.75.75 0 0 1-.75-.75z" clip-rule="evenodd"/></svg>';
+  hamburgerIcon.innerHTML = '<svg xmlns="http://www.w3.org/2000/svg" aria-hidden="true" viewBox="0 0 24 24" class="w-8 h-8 fill-current"><path fill-rule="evenodd" d="M3 6.75A.75.75 0 0 1 3.75 6h16.5a.75.75 0 0 1 0 1.5H3.75A.75.75 0 0 1 3 6.75zM3 12a.75.75 0 0 1 .75-.75h16.5a.75.75 0 0 1 0 1.5H3.75A.75.75 0 0 1 3 12zm0 5.25a.75.75 0 0 1 .75-.75h16.5a.75.75 0 0 1 0 1.5H3.75a.75.75 0 0 1-.75-.75z" clip-rule="evenodd"/></svg>';
```
Comment (rgravitvl) [inline blocks/header/header.js:461]: use Icon from icon.svg

**Review by rgravitvl — COMMENTED** [review]
Use all Icon form Icon SVG

**Pull request merged**